### PR TITLE
[Doc Improvement][Add a note for debugging supported in devices][2760360]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/calls-and-meetings/real-time-media-concepts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b1270f0d26cf9ada9727cbbef71d66d82ac1cdcc/msteams-platform%2Fbots%2Fcalls-and-meetings%2Freal-time-media-concepts.md) | 93 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/10fa8912-18fb-4a18-863e-de83dea9bd98) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/test-app-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3e5f2a7954335737a2901d6791de62c04a153757/msteams-platform%2Fconcepts%2Fbuild-and-test%2Ftest-app-overview.md) | 92 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/20d0cede-df28-4055-a18b-775bea591de4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/test-app-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4ed8752df29c306c43401132b8d1aeffdd76ef97/msteams-platform%2Fconcepts%2Fbuild-and-test%2Ftest-app-overview.md) | 92 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f5211d49-7489-4afd-bc0d-c81daa628a7f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/test-app-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4f2f4a718b6aeaf18ebae1e483b118655475b54f/msteams-platform%2Fconcepts%2Fbuild-and-test%2Ftest-app-overview.md) | 92 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b70b3748-56c4-4749-8200-511eb7cf4081) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/test-app-overview.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ Test your bot locally. For more information, see [debug your bot locally with an
 
 To view the console logs, view or modify html, css, and network requests during runtime, add breakpoints to your JavaScript code, and perform interactive debugging access the DevTools. For more information, see [access the DevTools for Teams tabs](~/tabs/how-to/developer-tools.md).
 
+> [!NOTE]
+> Debugging is supported in desktop, web, and android devices but not in iOS devices.
```
Comment (ashishguptaiitb) [inline msteams-platform/concepts/build-and-test/test-app-overview.md:29]: @v-npaladugu , it is supported `on` the devices and not `in` them. 
Also, it should be `Andriod`.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/test-app-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/43e91ce9b105028d5df821b983423e4e60caebe9/msteams-platform%2Fconcepts%2Fbuild-and-test%2Ftest-app-overview.md) | 92 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b47f1707-ed7c-496a-88ed-1d3953891bda) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by ashishguptaiitb — APPROVED** [review]
lgtm

**Pull request merged**